test/tests/functional/pbs_resource_usage_log.py

```diff
@@ -55,18 +55,27 @@ def setUp(self):
         attr2 = {'resources_available.mem': '200gb'}
         self.server.manager(MGR_CMD_SET, NODE, attr2, id=self.mom.shortname)
 
+    def cleanup_eatcpu(self, scripts):
+        for script in scripts:
+            cmd = 'pgrep -f ' + script
+            ret = self.du.run_cmd(cmd=cmd, level=logging.DEBUG)
+            for pid in ret['out']:
+                cmd = 'sudo -u ' + str(TEST_USER) + ' kill -9 ' + pid
```

Comment:
No need for sudo.  Use the runas argument.

Reply:
done